org.eclipse.winery.repository.ui/src/app/instance/sharedComponents/interfaces/interfaces.component.ts

```diff
@@ -371,9 +371,11 @@ export class InterfacesComponent implements OnInit {
     private handleGeneratedArtifact(response: Response) {
         let message = '';
         if (this.toscaType === ToscaTypes.NodeType) {
+            const artifacttemplateUrl = response.url.replace(backendBaseURL, '');
             message = 'It\'s available for download at ' +
-                '<a style="color: black;" href="' + response.headers.get('Location') + '">'
-                + this.implementation.name
+                '<a style="color: black;" href="' + hostURL + artifacttemplateUrl + '">'
```

Comment:
If you want to point to the UI URL, you don't need the `hostURL`, `'/#/' + artifacttemplteUrl` should suffice